website/src/playground/PlaygroundLoader.tsx

```diff
@@ -343,7 +343,7 @@ function initState(
 				searchParams.get("bracketSpacing") === "true" ||
 				defaultPlaygroundState.settings.bracketSpacing,
 			bracketSameLine:
-				searchParams.get("bracketSameLine") === "true" ??
+				searchParams.get("bracketSameLine") === "true" ||
```

Comment:
Vite was complaining and throwing an warning about this line where the left operand would always be used, might be a false alarm from it, so that's something we can change back, probably.

Reply:
We should revert this change, it previously caused some issue